ncl/ncl_index/ncl-index-table.csv

```diff
@@ -11,15 +11,26 @@ NCL Function,Description,Python Equivalent,Notes
-`abs <https://www.ncl.ucar.edu/Document/Functions/Built-in/abs.shtml>`__,"Returns the absolute value of numeric data","``abs()`` or ``numpy.abs()``",`example notebook <../ncl_entries/general_applied_math.ipynb#abs>`__
+`abs <https://www.ncl.ucar.edu/Document/Functions/Built-in/abs.shtml>`__,"Returns the absolute value of numeric data","``abs()`` or ``numpy.abs()``",`example notebook <../ncl_entries/general_applied_math.ipynb#abs-fabs>`__
 `avg <https://www.ncl.ucar.edu/Document/Functions/Built-in/avg.shtml>`__,"Computes the average of a variable regardless of dimensionality","``numpy.average()`` or ``numpy.mean()``",`example notebook <../ncl_entries/general_applied_math.ipynb#avg>`__
 `cumsum <https://www.ncl.ucar.edu/Document/Functions/Built-in/cumsum.shtml>`__,"Calculates the cumulative sum","``numpy.cumsum()``",`example notebook <../ncl_entries/general_applied_math.ipynb#cumsum>`__
-`decimalPlaces <https://www.ncl.ucar.edu/Document/Functions/Contributed/decimalPlaces.shtml>`__,"Truncates or rounds to the number of decimal places specified","``round()`` or ``numpy.round()``",`example notebook <../ncl_entries/general_applied_math.ipynb#decimalPlaces>`__
-`get_d2r <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_d2r.shtml>`__,"Return a constant that converts degrees to radians","``math.pi / 180``",`example notebook <../ncl_entries/general_applied_math.ipynb#get-d2r>`__
-`get_r2d <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_r2d.shtml>`__,"Return a constant that converts radians to degrees","``180 / math.pi``",`example notebook <../ncl_entries/general_applied_math.ipynb#get-r2d>`__
-`get_pi <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_pi.shtml>`__,"Return pi as a type float or double","``math.pi``",`example notebook <../ncl_entries/general_applied_math.ipynb#get-pi>`__
-`log <https://www.ncl.ucar.edu/Document/Functions/Built-in/log.shtml>`__,"Computes the natural log of a numeric type","``math.log()``",`example notebook <../ncl_entries/general_applied_math.ipynb#log>`__
-`log10 <https://www.ncl.ucar.edu/Document/Functions/Built-in/log10.shtml>`__,"Computes the log base 10 of a numeric type","``math.log10()``",`example notebook <../ncl_entries/general_applied_math.ipynb#log10>`__
+`decimalPlaces <https://www.ncl.ucar.edu/Document/Functions/Contributed/decimalPlaces.shtml>`__,"Truncates or rounds to the number of decimal places specified","``round()`` or ``numpy.round()``",`example notebook <../ncl_entries/general_applied_math.ipynb#decimalplaces-round>`__
+`get_d2r <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_d2r.shtml>`__,"Return a constant that converts degrees to radians","``math.pi / 180`` or ``numpy.pi / 180``",`example notebook <../ncl_entries/general_applied_math.ipynb#get-d2r>`__
```

Comment:
I can add it, but the confusion is that `get_d2r` doesn't actually convert degrees to radians, but it returns a constant that you can use to convert between them. [`get_d2r`](https://www.ncl.ucar.edu/Document/Functions/Contributed/get_d2r.shtml) returns a constant and is not a function

Reply:
Should I replace the NCL's function that returns a constant to do the translation with the function that does the translation, by replacing `np.pi / 180` with `np.deg2rad()`? It doesn't mimic NCL behavior, but it is what I imagine the purpose of this constant is for

Reply:
Ah, right.  I forgot this was set up as constant in NCL rather than a function.  

I think it still probably makes sense to at least mention it, but I'll leave it up to you.

Reply:
Will do!